cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -321,6 +321,49 @@
 }
 
 
+// Center content and bleed full-width images into gutters
+// Used on our story pages
+.content__center-image-bleed {
+
+    & > section {
```

Comment:
This would be `& .content_main > section {` if `content__center-image-bleed` is moved up to `content`.

Reply:
We should probably give `section` a name [in the template](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/_layouts/content-base-main-only.html#L7), like `content_main-inner` or something. Maaaaaybe something for another PR. Your call.

Reply:
> We should probably give `section` a name [in the template](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/_layouts/content-base-main-only.html#L7), like `content_main-inner` or something. Maaaaaybe something for another PR. Your call.

Sounds good to me. Seems harmless and helpful.

Reply:
Added the new class in  https://github.com/cfpb/consumerfinance.gov/commit/e9f1d87096db0a1e546dadc7b3f2dc17c0bd8d2f